src/directives/resizable/ResizableTable.ts

```diff
@@ -20,6 +23,9 @@ export interface ResizableTableOptions {
     handler: string; // class applied to col handler
   };
   columns: string[] | number[]; // the columns associated to cols handlers
+  charsPerCol: number; // The actual number of chars displayed per col
+  maxCharsPerCol: number; // The max number of chars to display per col
```

Comment:
I'm not quite sure if I understand what `charPerCol` is, "actual number of chars" is a little vague (find a better name and/or improve the comment ?).
Is is the number of chars displayed on first render before we correct it if the text overflows ?
Also where does the 7.5 default value come from ?

Reply:
Yes that is the chars render at first display to be able to calculate a ratio between label first width and number of chars display, that will give use the pixelPerChar that we can apply on the full chars length to get the final width